src/Aquifer.Jobs/appsettings.json

```diff
@@ -0,0 +1,16 @@
+{
+    "Email": {
+        "Marketing": { },
+        "Transactional": {
+            "SendResourceAssignmentNotification": {
+                "TemplateId": "sendgrid-id-here"
```

Comment:
Will it work to specify the template IDs here and not override them in the environment specific settings?  There's no current need for environment specific overrides.

Reply:
It will, we can move these if we want to. I wasn't sure where we would want them. I'll relocate them